tools/donate-cpu.py

```diff
@@ -221,15 +222,15 @@ def scanPackage(workPath, cppcheck, jobs):
     print('Analyze..')
     os.chdir(workPath)
     libraries = ' --library=posix'
-    if hasInclude('temp', '<wx/') or hasInclude('temp', '"wx/'):
+    if hasInclude('temp', ['<wx/', '"wx/']):
         libraries += ' --library=wxwidgets'
-    if hasInclude('temp', '<QString>'):
+    if hasInclude('temp', ['<QString>', '<QtWidgets>', '<QtGui/']):
```

Comment:
Could one use a regex `#Qt[A-Z][a-z]+>` instead?

Reply:
I thought about regexp usage. But IMHO it brings not much benefit, at least for now.